fec.config.js

```diff
@@ -0,0 +1,23 @@
+
+module.exports = {
+    appUrl: '/insights/malware',
+    debug: true,
+    useProxy: true,
+    proxyVerbose: true,
+    interceptChromeConfig: false,
+    plugins: [],
+    hotReload: process.env.HOT === 'true',
+    moduleFederation: {
+        exclude: ['react-router-dom'],
```

Comment:
Is this not already injected by default and we only need to add this federation config if the app would still run react router v5?

Reply:
You are correct, I will remove this

Reply:
https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1704901868733629